sprint_1/src/laura/uebungsprojekt/aufgaben/UPLK83_Pasch.java

```diff
@@ -0,0 +1,20 @@
+package laura.uebungsprojekt.aufgaben;
+
+import java.util.Random;
+
+public class UPLK83_Pasch {
+    public static boolean diceRoll(int dice_one, int dice_two, int dice_three) {
```

Comment:
Der Name der Funktion ist eigentlich nicht ganz richtig, da hier nicht die Würfel gerollt werden, sondern nur die bereits geworfenen Würfel ausgewertet werden. Besser `isPasch` oder generischer auf Englisch `areAllNumbersEqual`?

Vielleicht ist es aber trotzdem keine schlechte Idee, eine Funktion zu haben, die die Würfel wirft (statt es "per Hand" in der `.main()` zu machen)